packages/salesforcedx-vscode-core/src/commands/forceGenerateFauxClasses.ts

```diff
@@ -120,8 +121,7 @@ export class ForceGenerateFauxClassesExecutor extends SfdxCommandletExecutor<{}>
     if (refreshSource !== SObjectRefreshSource.Manual) {
       logName += '_' + refreshSource;
     }
-    this.logMetric(logName);
-
+    this.logMetric(logName, startTime);
```

Comment:
@lcampos Is there a reason why this instance of `this.logMetric` isn't inside of a listener for ` execution.processExitSubject()`?

Reply:
Forgot to refactor this piece, thanks for pointing it out.